File: 04.SQLite/MyNotesApp/app/src/main/java/com/dicoding/mynotesapp/db/DatabaseContract.java
package com.dicoding.mynotesapp.db;

import android.provider.BaseColumns;

/**
 * Created by dicoding on 10/12/2017.
 */

public class DatabaseContract {

    static final class NoteColumns implements BaseColumns {
        static String TABLE_NAME = "note";

        //Note title
        static String TITLE = "title";
        //Note description
        static String DESCRIPTION = "description";
        //Note date
        static String DATE = "date";

    }
}

File: SQLite/MyNotesApp/kotlin/app/src/main/java/com/dicoding/picodiploma/mynotesapp/db/DatabaseContract.kt
package com.dicoding.picodiploma.mynotesapp.db

import android.provider.BaseColumns

/**
 * Created by dicoding on 10/12/2017.
 */

internal class DatabaseContract {

    internal class NoteColumns : BaseColumns {
        companion object {
            const val TABLE_NAME = "note"

            //Note id
            val _ID = "_id"
            //Note title
            const val TITLE = "title"
            //Note description
            const val DESCRIPTION = "description"
            //Note date
            const val DATE = "date"
        }

    }
}
